### ISSUE TITLE
Sorting Movie releases with FS97 (Ex.) sorts into tvshows

### ISSUE THREAD

**Issue by overtoneblue** [itsToggle/rclone_RD#45]
![image](https://github.com/itsToggle/rclone_RD/assets/93581805/68c0bf2d-0ed0-4b12-97fc-1cd2d2741b89)

is sorted into tvshows, im thinking likely because of the FS96 or any other numbers after FS, for example joy has other torrents with FS87, etc etc. not sure if there could be a rule to get around this issue, but given that these are fairly common I think it was a useful bug to officially track here. Thanks for all your work :)

**Comment by Lukkie** [comment]
Same for `Eragon.(2006).BDRip.1080p.[denis100].mkv`

**Comment by overtoneblue** [comment]
yeah, may need to slowly add the edge cases to sorting rules. I'm not familiar with the best way to do this so i'll leave it for someone more experienced to comment here until i can look into it further and test. Obviously we dont want to accidentally be sorting tvshows back into movies. 

Updating (more edge cases): 
The.Foreigner.(2017).BDRip-1080p.MegaPeer.(qqss44).mkv
Ferris Buellers Day Off 1986 (2160p x265 10bit FS89 Joy).mkv

**Comment by CedricPelchat** [comment]
Hey, I just want to add more edge cases. [ReinForce] Shiki (BDRip 1920x1080 x264 FLAC)  is a show that goes into movies. Here are other cases that went into movies for reasons I don't know:

Todd.McFarlane's.Spawn.1997.WEB-DL.1080_[teko]
bbc-connections-1978
Dekalog 1989 1080p Remux AVC FLAC 1.0
Futurama 1999 1080p

Those went into default:
[itaita] Welcome to the N.H.K. (ITA Dub) [1080p AMZN WEB-DL]
[Team246] Dorohedoro - integral [bd-remux customs 1080p Multi pcm][vostfr]
Ayakashi - Samurai Horror Tales [BD 1080p x265 FLAC] [Dual Audio]
Cowboy Bebop The Ultimate Enthusiast's Collected Edition
FLCL
Fullmetal Alchemist (1080p) (Debanded)
Fullmetal Alchemist Brotherhood + OVAs + Specials [BD 1080p Hi10 FLAC] [Dual Audio]
The Hunger Games Quadrilogy.UHD BDRemux 2160p Dolby Vision
To Be Hero

Hoping this can help. Is there a way I can manually ignore folders from a plex library? Normally there would be the .plexIgnore file, but I can't add that to my RD folder, in rclone, it getrs instantly deleted.
